src/EventStore.Client/EventStoreClientConnectivitySettings.cs

```diff
@@ -76,10 +76,15 @@ public class EventStoreClientConnectivitySettings {
 		public bool IsSingleNode => GossipSeeds.Length == 0;
 
 		/// <summary>
-		/// True if communicating over a secure channel; otherwise false.
+		/// False if communicating over a secure channel; otherwise true.
 		/// </summary>
 		public bool Insecure { get; set; }
 
+		/// <summary>
+		/// True if certificates will be validated; otherwise false. 
+		/// </summary>
+		public bool TlsVerifyCert { get; set; }
```

Comment:
```suggestion
		public bool TlsVerifyCert { get; set; } = true;
```

should default to true?